### PR TITLE
add: Azure Container Apps detectors

### PR DESCRIPTION
**Pull request by maxpoullain**
Azure Container Apps detectors: 
- **critical** _heartbeat_ (based on memory)
- **warning** _restarts count_

### CODE REVIEW

**Review by haedri — APPROVED** [review]

File: modules/integration_azure-container-apps/conf/00-heartbeat.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module: Azure Container Apps
+name: heartbeat
+filtering: "filter('resource_type', 'Microsoft.App/containerApps') and filter('aggregation_type', 'minimum')"
+
+transformation: true
+aggregation: ".mean(by=['azure_resource_id'])"
```
Comment (jmapro) [inline modules/integration_azure-container-apps/conf/00-heartbeat.yaml:6]:
```suggestion
aggregation: ".mean(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region', 'azure_tag_env'])"
```

File: modules/integration_azure-container-apps/conf/02-replicas.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+module: Azure Container Apps
+name: replicas
+filtering: "filter('resource_type', 'Microsoft.App/containerApps') and filter('aggregation_type', 'count')"
+
+transformation: true
+aggregation: false
```
Comment (jmapro) [inline modules/integration_azure-container-apps/conf/02-replicas.yaml:6]: no aggregation by name or something like that ?

File: modules/integration_azure-container-apps/conf/01-restarts.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+module: Azure Container Apps
+name: restarts
+filtering: "filter('resource_type', 'Microsoft.App/containerApps')"
+
+transformation: ".sum(over='15m')"
+aggregation: ".sum(by=['podname'])"
```
Comment (jmapro) [inline modules/integration_azure-container-apps/conf/01-restarts.yaml:6]: Only podname ? What if multiple ACA ?

**Review by jmapro — CHANGES_REQUESTED** [review]

File: modules/integration_azure-container-apps/conf/00-heartbeat.yaml (hunk 1 above)
Comment (BzSpi) [reply modules/integration_azure-container-apps/conf/00-heartbeat.yaml:6]: No, `azure_tag_env` is not generic. It should be in templates, not module.

File: modules/integration_azure-container-apps/conf/00-heartbeat.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+signals:
+  signal:
+    metric: "WorkingSetBytes"
+
+rules:
+  critical:
+    threshold: 0
+    comparator: "="
+    lasting_duration: 5m
```
Comment (BzSpi) [inline modules/integration_azure-container-apps/conf/00-heartbeat.yaml:16]: No condition needed for heartbeat.
```suggestion
rules:
  critical:
```

File: modules/integration_azure-container-apps/conf/02-replicas.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+module: Azure Container Apps
```
Comment (BzSpi) [inline modules/integration_azure-container-apps/conf/02-replicas.yaml:1]: I do not understand the purpose of this detector.

**Review by BzSpi — CHANGES_REQUESTED** [review]

**Pull request merged**